deploy/crontab-openprescribing

```diff
@@ -1,6 +1,6 @@
 # min hour day/month month day/week
 
-00 01 * * * hello /webapps/openprescribing/deploy/run_pipeline_e2e_tests.sh
+00 01 * * * hello /webapps/openprescribing/deploy/run_pipeline_e2e_tests.sh >/tmp/e2e-$(date +%Y-%m-%d).out
```

Comment:
As we are in the shell I would use `mktemp` as this respects TMPDIR and is better WRT race conditions, e.g.

   ls > $(mktemp -t e2e-$(date +%Y-%m-%d)-XXXX.log)

Reply:
Nifty, thanks.